homeassistant/components/device_tracker/__init__.py

```diff
@@ -486,7 +486,7 @@ def get_vendor_for_mac(self):
             return None
 
         # prevent lookup of invalid macs
-        if not len(self.mac.split(':')) == 6:
+        if not len(self.mac) == 17:
```

Comment:
My guess would be this is the part that blows up: 

https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/components/device_tracker/__init__.py#L492-L497